elyra/pipeline/registry.py

```diff
@@ -161,11 +161,18 @@ def get_common_config(self, config_name):
         return common_json
 
     def _get_component_catalog_json(self):
-        # then sys.prefix, where installed files will reside (factory data)
-        catalog_dir = os.path.join(jupyter_core.paths.ENV_JUPYTER_PATH[0], self._components_dir)
-        catalog_file = os.path.join(catalog_dir, f"{self._type}_component_catalog.json")
-        with open(catalog_file, 'r') as f:
-            catalog_json = json.load(f)
+        try:
+            # then sys.prefix, where installed files will reside (factory data)
+            catalog_dir = os.path.join(jupyter_core.paths.ENV_JUPYTER_PATH[0], self._components_dir)
+            catalog_file = os.path.join(catalog_dir, f"{self._type}_component_catalog.json")
+            with open(catalog_file, 'r') as f:
+                catalog_json = json.load(f)
+        except FileNotFoundError:
+            self.log.error(f"The component catalog could not be found at path '{catalog_file}'.")
+            raise FileNotFoundError(f"The component catalog could not be found at path '{catalog_file}'.")
+        except Exception:
+            self.log.error(f"The component catalog at path '{catalog_file}' could not be loaded.")
```

Comment:
should log exception text to enable troubleshooting

Reply:
Good point, thank you!